src/Adapter/Platform/PlatformInterface.php

```diff
@@ -56,8 +56,8 @@ public function quoteValue($value);
      *
      * The ability to quote values without notices
      *
-     * @param $value
-     * @return mixed
+     * @param scalar $value
```

Comment:
Not really a BC break, but we should avoid these changes in minor releases

Reply:
The way it was written before, it was essentially missing, though. Should I have said `mixed` instead? I was trying to make these align to what the code actually expects...